web/src/lib/client/storage.js

```diff
@@ -59,4 +64,21 @@ export default class StorageClient extends Client {
       CandidateDevices: cockpit.variant("as", candidateDevices)
     });
   }
+
+  /**
+   * Register a callback to run when properties in the Actions object change
+   *
+   * @param {function} handler - callback function
+   */
+  onActionsChange(handler) {
+    return this.onObjectChanged(ACTIONS_PATH, changes => {
+      const newActions = changes.All.v.map(action => {
+        const { Text: textVar, Subvol: subvolVar } = action.v;
+        return { text: textVar.v, subvol: subvolVar.v };
+      });
+      handler({ All: newActions });
```

Comment:
why not pass just new actions as parameter?

Reply:
If we add a new property in the future, we would need to update all the handlers to cope with the change. Passing the actions under the `All` property makes this code future-proof.